darts/models/filtering/moving_average_filter.py

```diff
@@ -7,7 +7,7 @@
 from darts.timeseries import TimeSeries
 
 
-class MovingAverage(FilteringModel):
+class MovingAverageFilter(FilteringModel):
```

Comment:
I am not sure to understand why it is necessary to rename this filtering model?

Reply:
In the issue and the original [PR](https://github.com/unit8co/darts/pull/1201) one of the points was that "MovingAverage" might be a bit misleading for users who might mistake it for the new prediction model. When taking over the issue I thought about it and decided to do the same, but if you think it's not worth the trouble I'll change it back

Reply:
My bad, I was not aware of this conversation, `MovingAverageFilter` it is then!